Resources/Private/Assets/JavaScript/MediaUpload.js

```diff
@@ -95,10 +94,9 @@
 								.find('.view-btn')
 								.each(function(index, element) {
 									var uri = $(element).attr('href');
-									var basename = uri.replace("/typo3temp/MediaUpload/", "");
+									var basename = uri.replace("/typo3temp/pics/", "");
```

Comment:
I have reverted the path to typo3temp/MediaUpload as there is a cleanup action within EXT:media_upload that will potentially and recursively delete all files in this folder (=> would have been typo3temp/pics)